man/coredumpctl.xml

```diff
@@ -227,11 +235,21 @@
         <option>--output=</option>. </para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><command>debug</command></term>
+
+        <listitem><para>Invoke the a debugger on the last core dump
+        matching specified characteristics. The debugger used (either gdb or lldb)
+        will first be read from <option>--debugger</option>, then from
+        <varname>$SYSTEMD_DEBUGGER</varname>. If neither is given, then the
+        default of gdb will be used.</para></listitem>
+      </varlistentry>
+
       <varlistentry>
         <term><command>gdb</command></term>
 
-        <listitem><para>Invoke the GNU debugger on the last core dump
-        matching specified characteristics. </para></listitem>
+        <listitem><para>An alias for <command>debug</command> that always
+        uses gdb.</para></listitem>
```

Comment:
I am pretty sure `gdb` should be retained as compat alias only. That means it should behave exactly like `debug`, and not appear in the man page or `--help` text anymore.